README.md

```diff
@@ -74,6 +74,7 @@ Notes:
 * You need to have a clean working copy of your registry.
 * The package must be stored as a git working copy, e.g. having been
   cloned with `Pkg.develop`.
+* Currently, relative paths of packages [are not supported](https://github.com/JuliaLang/Pkg.jl/issues/677) by Pkg. Make sure to have `dev`'d an absolute path.
```

Comment:
How does it make a difference whether you `dev` an absolute path?

Reply:
You are right, I must have brainfarted.